src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -646,19 +646,7 @@ boolean checkTransportFilter(final AuditCategory category, final String action,
         }
 
         //skip internals
-        if(action != null
-                &&
-                ( action.startsWith("internal:")
-                        || action.startsWith("cluster:monitor")
-                        || action.startsWith("indices:monitor")
-                )
-        ) {
-
-
-            //if(log.isTraceEnabled()) {
-            //    log.trace("Skipped audit log message due to category ({}) or action ({}) does not match", category, action);
-            //}
-
+        if (action != null && action.startsWith("internal:")) {
```

Comment:
What are the actions under internal: ?

Reply:
Internal actions seem to be used for core ES functionality only and not for user generated actions. Here are some of the internal actions

- internal:coordination/fault_detection/leader_check
- internal:coordination/fault_detection/follower_check
- internal:cluster/coordination/join
- internal:cluster/coordination/join/validate
- internal:cluster/coordination/start_join
- internal:cluster/request_pre_vote
- internal:cluster/coordination/publish_state
- internal:cluster/coordination/commit_state
- internal:cluster/node/mapping/refresh
- internal:cluster/shard/started
- internal:cluster/shard/failure
- internal:admin/tasks/ban
- internal:index/shard/recovery/start_recovery
- internal:index/shard/recovery/reestablish_recovery
- internal:index/shard/recovery/filesInfo
- internal:index/shard/recovery/file_chunk
- internal:index/shard/recovery/clean_files
- internal:index/shard/recovery/translog_ops
- internal:index/shard/recovery/prepare_translog
- internal:index/shard/recovery/finalize
- internal:index/shard/recovery/handoff_primary_context
- internal:index/shard/exists
- internal:cluster/nodes/indices/shard/store
- internal:tcp/handshake
- internal:transport/handshake
- internal:transport/proxy/
- internal:cluster/snapshot/update_snapshot_status
- internal:admin/repository/verify
- internal:indices/admin/upgrade
- internal:indices/flush/synced/pre
- internal:index/seq_no/resync
- internal:gateway/local/started_shards
- internal:gateway/local/meta_state
- internal:gateway/local/allocate_dangled